packages/ui/src/theme/createTheme/createComponentCSS.ts

```diff
@@ -58,17 +58,21 @@ function recursiveComponentCSS(baseSelector: string, theme: BaseTheme) {
   return str;
 }
 
+interface CreateComponentCSSProps {
```

Comment:
Nit: `Props` is a React specific naming construct and may lead to consumer confusion when using util functions that are not React components, for functions interfaces prefer `Input` or `Params`

Reply:
Good catch